grakn-client/src/main/java/ai/grakn/remote/RemoteGraknTx.java

```diff
@@ -178,17 +199,18 @@ public QueryBuilder graql() {
 
     @Override
     public void close() {
-        client.close();
+        if(!isClosed()) client.close();
     }
 
     @Override
     public void abort() {
-        throw new UnsupportedOperationException(); // TODO
+        if(!isClosed()) client.close();
```

Comment:
Is `abort` always the same as `close`? Sounds like a `default` implementation :D

Reply:
yep it is and that is a good suggestion!